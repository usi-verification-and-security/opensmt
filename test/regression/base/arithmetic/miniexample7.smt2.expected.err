terminate called after throwing an instance of 'opensmt::LANonLinearException'
  what():  Term (* -1 (div x y)) is non-linear
Aborted
