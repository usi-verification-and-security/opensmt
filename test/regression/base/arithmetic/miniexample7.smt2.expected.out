Term (div x y) is non-linear
unknown
